ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/PartyManagementServiceIT.scala

```diff
@@ -7,8 +7,10 @@ import com.daml.ledger.api.testtool.infrastructure.Allocation._
 import com.daml.ledger.api.testtool.infrastructure.LedgerTestSuite
 import com.daml.ledger.api.v1.admin.party_management_service.PartyDetails
 import com.daml.ledger.client.binding
+import com.daml.ledger.test_stable.Test.Dummy
```

Comment:
```suggestion
import com.daml.ledger.test.Test.Dummy
```
This changed in between when you first made the PR and now.

Reply:
Correct, this is needed to get the test to compile